janmatyoo/ev-employee-onboarding-exercise
# PR for EVOBE-156, EVOBE-167

### Ticket
<https://phunts.atlassian.net/browse/EVOBE-156>
<https://phunts.atlassian.net/browse/EVOBE-167>


### Changes
- Containerize Backend
- Containerize Frontend
- Added docker-compose to run multiple containers

### Steps to Test
1. Go to the root folder
2. Run the cli command `docker-compose up`
3. You should now see the website at `http://localhost:3000/`

### Resources
- N/A
